bundles/binding/org.openhab.binding.expire/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
You have .settings in your .gitignore, but this PR contains files from your .settings directory.  Most bindings don't have their own .gitignore.  Can you instead rely on the one higher in the tree?